factory/glideFactoryEntry.py

```diff
@@ -1295,7 +1306,8 @@ def unit_work_v3(
 
     if "scitoken" in auth_method:
         if os.path.exists(scitoken_file):
-            if token_util.token_file_expired(scitoken_file):
+            scitoken = create_credential(path=scitoken_file)
```

Comment:
Isn't the scitoken object already there from before? Or is the idea that is re-created from file all the times that it is needed?

Reply:
Updated.